src/filesink.cpp

```diff
@@ -67,7 +67,7 @@ namespace g3 {
 
       _write_buffer.append(data);
       if (++_write_counter % _write_to_log_every_x_message == 0) {
-         out << message.get().toString(_log_details_func) << std::flush;
+         out << _write_buffer << std::flush;
```

Comment:
damn, this one was ugly. thanks for catching it so quickly.

Reply:
👍, pleasure contributing to this already great piece of code !